### The data structure to associate a door with his location

string location   
string door   